### PR TITLE
postCSS

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "neomad-styleguide",
+  "version": "1.0.0",
+  "description": "Neomad styleguide, built with atomic design mindset",
+  "scripts": {
+    "dev": "postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --watch",
+    "prod": "postcss ./content/styles/main.css -o ./content/dist/main.css"
```
Comment (vinyll) [inline package.json:7]: Why calling it _styleguide.css_ in dev and _main.css_ in prod?

**Review by newick — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (fuentesloic) [reply package.json:7]: styleguide.css is build only for styleguide repo.
It's build from stylguide wich import main.css

main.css is build for neomad repo (and without styleguide structure).

If you think it’s weird or not logic, we could change.
Do you prefer something like dev.css and prod.css ??

Comment (newick) [reply package.json:7]: my idea was to have a _styleguide_ dedicated file only in the styleguide repo and _main_ or whatever the name used in both repos

Comment (fuentesloic) [reply package.json:7]: ok works for me!

Comment (fuentesloic) [reply package.json:7]: @newick ok for _styleguide.css_ but about _main.css_ (whatever the name ^^) just one call do by neomad.org could be better, so just one file could be avoid us different version and keep all the CSS here

**Review by vinyll — APPROVED** [review]
Seems clear to me 👌

**Pull request merged**